server/models/Request.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose');
+
+const requestSchema = new mongoose.Schema({
+    user_id: {
```

Comment:
Maybe add a reference to the User/Profile model (here and below) based on our use cases?